packages/orchestrator-ui-components/src/components/WfoTree/treeUtils.spec.ts

```diff
@@ -119,3 +120,239 @@ describe('getWfoTreeNodeDepth', () => {
+                                    "description": "KLM SP AH001A 10 Gbit/s"
+                                },
+                                "parent": 7,
+                                "productBlockInstanceValues": [
+                                ],
+                                "subscriptionInstanceId": "a05d89ad-9300-4a05-b7f7-0a8d78dd2a07",
+                                "inUseByRelations": [
+                                ],
+                                "icon": "",
+                                "label": "SP 10G tagged (ah001a-jnx-test-floris-1-vtb)",
+                                "children": [
+                                    {
+                                        "id": 9,
+                                        "subscription": {
+                                            "subscriptionId": "00e4fbbb-9192-4fac-9066-5d1381c5f0a4",
+                                            "description": "Node ah001a-jnx-test-floris-1-vtb Ah001A"
+                                        },
+                                        "parent": 8,
+                                        "productBlockInstanceValues": [
+                                        ],
+                                        "subscriptionInstanceId": "6ff2d010-9e16-44ed-ac62-b36b9438c7ff",
+                                        "inUseByRelations": [
+                                        ],
+                                        "icon": "tokenConstant",
+                                        "label": "Node ah001a-jnx-test-floris-1-vtb",
+                                        "children": [],
+                                        "isOutsideCurrentSubscription": true
+                                    }
+                                ],
+                                "isOutsideCurrentSubscription": true
+                            }
+                        ],
+                        "isOutsideCurrentSubscription": false
+                    }
+                ],
+                "isOutsideCurrentSubscription": false
+            },
+            {
+                "id": 1,
+                "subscription": {
+                    "subscriptionId": "556facd3-6399-4c76-aeb1-114849da2f96",
+                    "description": "AMPATH L3VPN AH001A 215 Mbit/s"
+                },
+                "parent": 0,
+                "productBlockInstanceValues": [
+                ],
+                "subscriptionInstanceId": "43fc68ff-8644-46b1-9e8f-bb4bb46608b9",
+                "inUseByRelations": [
+                ],
+                "icon": "tokenConstant",
+                "label": "L3VPNSS AS1105",
+                "children": [],
+                "isOutsideCurrentSubscription": false
+            }
+        ],
+        "isOutsideCurrentSubscription": false
+    }`;
+
+    const tree = JSON.parse(json) as TreeBlock;
+
+    it('should return 1 for the root node', () => {
```

Comment:
Looks a bit verbose

Reply:
The test data is an actual data set that showed me wrong behaviour. I wanted to make sure it passed completely. I think it doesn't hurt to have it here since its there already